### PR TITLE
remove Driver from docs and replace with SDK

### PR DESCRIPTION
**Pull request by elof**
This PR removes the term Driver from the docs and replaces them with SDK. Drivers are still mentioned in the example code on some pages because the term is hard-coded into functionally using the SDK in some instances, but replacing the terminology in the docs will make it more clear what they are and will add the term SDK to search

### CODE REVIEW

File: docs/c8ql/c8ql-tutorial.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ This small collection of 8 filming locations comes with two attributes, a *name*
 
 ### Create documents
 
-Before we can insert documents with C8QL, we need a place to put them in - a collection. Collections can be managed via the web interface, c8sh or a driver. It is not possible to do so with C8QL however.
+Before we can insert documents with C8QL, we need a place to put them in - a collection. Collections can be managed via the web interface, c8sh or SDK. It is not possible to do so with C8QL however.
```
Comment (diana-macrometa) [inline docs/c8ql/c8ql-tutorial.md]:
```suggestion
Before we can insert documents with C8QL, we need a place to put them in -- a collection. Collections can be managed via the web interface, c8sh, or SDK. It is not possible to do so with C8QL however.
```

File: docs/c8ql/common-errors.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ C8QL is not vulnerable to parameter injection in itself, but queries might be co
 
 Assembling query strings with simple **string concatenation** looks trivial, but is potentially **unsafe**. 
 
-You should use [bind parameters](../queryworkers/fundamentals.md#bind-parameters) instead whenever possible, use query building functionality if provided by a driver (see [Drivers](../essentials/Drivers.md) for example) or at least sanitize user input with great care.
+You should use [bind parameters](../queryworkers/fundamentals.md#bind-parameters) instead whenever possible, use query building functionality if provided by a SDK (see [SDKs](../essentials/SDKs.md) for example) or at least sanitize user input with great care.
```
Comment (diana-macrometa) [inline docs/c8ql/common-errors.md]: Please make sure that the file name is all lowercase.

File: docs/c8ql/index.md [hunk 3]
```diff
@@ -25,6 +25,6 @@ In its purpose, C8QL is similar to the Structured Query Language (SQL). C8QL sup
 
 It is a pure data manipulation language (DML), not a data definition language (DDL) or a data control language (DCL). The syntax of C8QL queries is different to SQL, even if some keywords overlap. Nevertheless, C8QL should be easy to understand for anyone with an SQL background.
 
-We have drivers for [JavaScript](https://github.com/Macrometacorp/jsC8) & [Python](https://github.com/Macrometacorp/pyC8)
+We have SDKs for [JavaScript](https://github.com/Macrometacorp/jsC8) & [Python](https://github.com/Macrometacorp/pyC8)
```
Comment (diana-macrometa) [inline docs/c8ql/index.md]:
```suggestion
We have SDKs for [JavaScript](https://github.com/Macrometacorp/jsC8) and [Python](https://github.com/Macrometacorp/pyC8)
```

File: docs/essentials/SDKs.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 sidebar_position: 6
-title: Drivers
-slug: drivers
+title: SDKs
```
Comment (diana-macrometa) [inline docs/essentials/SDKs.md:3]: Change the filename to all lowercase.

File: docs/essentials/SDKs.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ yarn add jsc8
 (or)
 npm install jsc8
 
-If you want to use the driver outside of the current directory, you can also install it globally using the `--global` flag:
+If you want to use the SDK outside of the current directory, you can also install it globally using the `--global` flag:
```
Comment (diana-macrometa) [inline docs/essentials/SDKs.md]:
```suggestion
If you want to use the SDK outside of the current directory, then you can also install it globally using the `--global` flag:
```

**Review by diana-macrometa — APPROVED** [review]
A bunch of small changes and one medium:

Please make sure that filenames are all lowercase.

File: docs/c8ql/c8ql-tutorial.md (hunk 1 above)
Comment (elof) [reply docs/c8ql/c8ql-tutorial.md]: I changed the case of the SDK filename @diana-macrometa

**Pull request merged**